sklearn/utils/tests/test_validation.py

```diff
@@ -362,3 +364,7 @@ def test_check_consistent_length():
     assert_raises_regexp(TypeError, 'estimator', check_consistent_length,
                          [1, 2], RandomForestRegressor())
     # XXX: We should have a test with a string, but what is correct behaviour?
```

Comment:
Please do not add `if __name__ == "__main__":` blocks to the test files. Instead you can use:

```
nosetests sklearn/utils/tests/test_validation.py:test_ordering
```